blockchain/block.go

```diff
@@ -28,6 +27,9 @@ type BlockHeader struct {
 	Time uint32
 	// Nonce starts at 0 and increments by 1 for every hash when mining
 	Nonce uint64
+	// ExtraData is an extra field that can be filled with arbitrary data to
+	// be stored in the block
+	ExtraData []byte
```

Comment:
This is what chad and i were talking about on slack, I think we decided to not limit any size for now. I made a ticket for it.